napari/utils/colormaps/colormap_utils.py

```diff
@@ -1,18 +1,25 @@
+import warnings
 from collections import OrderedDict
 from threading import Lock
-from typing import Dict, List, Tuple, Union
+from typing import Dict, List, Optional, Tuple, Union
 
 import numpy as np
 from vispy.color import BaseColormap as VispyColormap
-from vispy.color import get_colormap, get_colormaps
+from vispy.color import Color, ColorArray, get_colormap, get_colormaps
 
 from ..translations import trans
 from .bop_colors import bopd
 from .colormap import Colormap
+from .standardize_color import transform_color
 from .vendored import cm, colorconv
 
+# All parsable input color types that a user can provide
+ColorType = Union[List, Tuple, np.ndarray, str, Color, ColorArray]
```

Comment:
still hoping for the day that we move to `pydantic.types.Color` :)

Reply:
yes for sure - maybe @andy-sweet can make an issue for that if we don't have one already. i think can come in followup PRs though :-)